### DISCUSSION TITLE
S-132 printer addon board

### DISCUSSION THREAD

**Discussion by jrd25** [thehighnibble/thehighnibble.github.io#178] [S-132 add-on for IMSAI & Cromemco replicas]
This information may be somewhere on the s-132 site, but I can't find it.  The printer board plugs into the 40 pin connector on the s-132 main board.  Could someone let me know which way it is to be oriented.  There are two possibilities, and I don't want to burn something up by plugging it in the wrong way.  Thanks in advance for any assistance.

JRD25

**Comment by ronaldlw** [comment]
Based on some of the pin labels on the S-132 and the prototype board, it looks like the board of the plugin card should be to the left of the S-132 header.
![image](https://github.com/user-attachments/assets/4994fd33-4cb5-4794-8574-7739298c6a02)

![image](https://github.com/user-attachments/assets/681d5945-4e55-4848-b98c-00aecca405d2)

this seems to be verified by a screen capture from [ShadowTron Blog](https://www.youtube.com/@ShadowTronBlog) 's youtube video as well:
https://youtu.be/skwqI-xDGBM?si=esJSHEm0j5OU3qhr

![image](https://github.com/user-attachments/assets/0fe61adf-7f60-477f-8bda-b042c4da80cd)
